crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -666,7 +666,7 @@ fn get_common_mosquitto_config_file_path(
 // To confirm the connected c8y tenant is the one that user configured.
 fn check_connected_c8y_tenant_as_configured(tedge_config: &TEdgeConfig, configured_url: &str) {
     match get_connected_c8y_url(tedge_config) {
-        Ok(url) if url == configured_url => {}
+        Ok(url) if url == configured_url.rsplitn(2, ':').last().unwrap() => {}
```

Comment:
We can do this more easily by calling `.host()` before `.to_string()` [on line 143](https://github.com/thin-edge/thin-edge.io/blob/main/crates/core/tedge/src/cli/connect/command.rs#L143), then we don't have to unwrap

Reply:
Resolved by 465fd5fc1853df9e0f5c9e286c822bd14c44e42b

Reply:
As directed by @jarhodes314 I also removed the redundant `.clone()`.